### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t1903

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/66ec74e85f5c990/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/6c21f9bc16f7224/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/66ec74e85f5c990/output.txt

Total script time: 1.32 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/6c21f9bc16f7224/output.txt

Total script time: 0.70 mins

Server is running on http://pr972.butyr.org/templates/basic/
Run the Test Suite: http://pr972.butyr.org/test/

File: templates/assets/plugins/image/popcorn.image.js [hunk 1]
```diff
@@ -77,12 +77,20 @@
...
+          _link.classList.add( "image-plugin-link" );
 
           _image = document.createElement( "img" );
           _image.addEventListener( "load", function() {
             _image.classList.add( "image-plugin-img" );
 
-            ( _link || _container ).appendChild( _image );
+            _link.appendChild( _image );
+            _container.appendChild( _link );
+            if ( _image.getBoundingClientRect().height > _image.getBoundingClientRect().width  ) {
```
Comment (secretrobotron) [inline templates/assets/plugins/image/popcorn.image.js]: Cache the value of `getBoundingClientRect()`.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f593309d1311ae3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/076f2b2ffdc521c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f593309d1311ae3/output.txt

Total script time: 1.42 mins

Server is running on http://pr972.butyr.org/templates/basic/
Run the Test Suite: http://pr972.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/076f2b2ffdc521c/output.txt

Total script time: 1.47 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/aa799f0a57efa21/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/cd6ffebf58e1acf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/aa799f0a57efa21/output.txt

Total script time: 1.71 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cd6ffebf58e1acf/output.txt

Total script time: 1.70 mins

Server is running on http://pr972.butyr.org/templates/basic/
Run the Test Suite: http://pr972.butyr.org/test/

**Pull request merged**